geode-core/src/main/java/org/apache/geode/management/internal/cli/result/ResultBuilder.java

```diff
@@ -230,6 +230,13 @@ public static CommandResult fromJson(String json) {
         resultData = errorResultData;
       }
 
+      Integer statusCode = (Integer) jsonObject.get("status");
+      if (Result.Status.OK.getCode() == statusCode) {
```

Comment:
A bit of a nit-pick, but I would change this to `if (statusCode == Result.Status.OK.getCode()) {`. I think that conditionals written as `(actual <conditional> test-value)` easier to read.